script/jenkins/petals_regression_test/Jenkinsfile

```diff
@@ -14,7 +14,7 @@ pipeline {
         cp ~/jobs/petals_install_env/workspace/climada.conf climada.conf
         python script/jenkins/set_config.py test_directory ~/jobs/petals_install_env/workspace/climada_petals
 
-        pytest --junitxml=tests_xml/tests.xml ~/jobs/petals_install_env/workspace/climada_petals
+        PYTHONPATH=.:$PYTHONPATH pytest --junitxml=tests_xml/tests.xml ~/jobs/petals_install_env/workspace/climada_petals
```

Comment:
Part of this PR?

Reply:
If you need the current directory in your PYTHONPATH, call pytest with the python interpreter like so:
```
python -m pytest
```
See [Calling pytest through python -m pytest](https://docs.pytest.org/en/7.1.x/how-to/usage.html#calling-pytest-through-python-m-pytest)

However, this should not be necessary if both modules are installed

Reply:
No, not part of the PR, that is also in develop and right that's a substitute for `python -m pytest` (thanks @peanutfun for pointing this out, didn't know that)
And it's necessary, exactly _because_ both modules are installed. 😁

Reply:
I don't understand. If both are installed, the interpreter should have no trouble finding them. What are the contents of the `./` directory and why do you need to add it to the PYTHONPATH?

Reply:
'./' is the root directory of the repository. Putting it in the first place of the python path is meant to make sure any climada module is taken from here rather then from the climada_petals installation.
Without this `import climada` yields the climada module from the develop branch.
Note that the environment is not installed but just activated.

Reply:
I think I understand. Installing Petals will pull the pip version of Climada, but you want the local one, right? Then I suggest to call pytest from the interpreter, as discussed. Setting the Pythonpath manually seems like a "dirtier" option to me.

These changes are now gone somehow?